package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "start": "npm-run-all --parallel startBackendDev buildFrontendDev",
+    "startProd": "npm-run-all --parallel startBackendProd buildFrontendProd",
```

Comment:
running in parallel causes some issues, this was previously `npm-run-all -s` to run it in sequence before we got rid of the option and used a shell script in [this PR](https://github.com/HL7-DaVinci/dtr/pull/159/files).  Reinstating this option, I'd opt to use the `-s` flag instead.

Reply:
Though there's also some issues with using `npm-run-all` with docker that we've run into in the past as well, see [this PR](https://github.com/HL7-DaVinci/dtr/pull/157)

Reply:
I haven't been able to see any issues with running production in parallel or using npm-run-all within the docker containers